contracts/src/core/AlignedLayerServiceManager.sol

```diff
@@ -149,4 +182,12 @@ contract AlignedLayerServiceManager is
                 verificationDataBatchIndex
             );
     }
+
+    function balanceOf(address account) public view returns (uint256) {
+        return batchersBalances[account];
+    }
+
+    receive() external payable {
+        batchersBalances[msg.sender] += msg.value;
+    }
```

Comment:
This is already being done in `createNewTask()`. We should either remove this function, or add a function in the batcher contract to only send funds to this contract.
This is because now the funds mapping is to the batchers' contract, and not his wallet.
I think removing this function is the simpler solution